js/models.js

```diff
@@ -104,6 +104,31 @@ class StoryList {
       /*
     old code, might be useful at some point */
   }
+
+  /* Recieves a user obj and a storyId
+  *   -verifies story belongs to user
+  *   -deletes story from api
+  */
+  async deleteStoryFromList(user, storyId){
+    const belongsToUser = user.ownStories.some((story) => (story.storyId == storyId));
```

Comment:
Nice usage of `some()`! 😎

Reply:
Thanks, I would rather do that then check indexOf